db/migrate/20190605153707_add_complete_to_tslr_claims.rb

```diff
@@ -0,0 +1,5 @@
+class AddCompleteToTslrClaims < ActiveRecord::Migration[5.2]
+  def change
+    add_column :tslr_claims, :complete, :boolean
```

Comment:
Let's not use `complete` for this, as the claim still has some processing to be made before it's complete. Also, do we need a new field? Isn't completeness a derived property, based on the status of the other claim attributes? I think we can use `claim.valid?(:submit)` instead, which runs all the validators to establish if the claim is submittable.

Reply:
Yeah, that makes sense. Would save having to add another migration too.